website/content/docs/release-notes/v0_18_0.mdx

```diff
@@ -41,6 +41,18 @@ description: >-
     Learn more:&nbsp; <a href="#known-issues-and-breaking-changes">Known issues and breaking changes </a>
     </td>
   </tr>
+
+<tr>
+    <td style={{verticalAlign: 'middle'}}>
+    Go version 1.23.3 x509 key pair behavior changes
```

Comment:
I believe 18.0 uses go 1.23.1 (according to go.mod) and the behaviour was introduced with Go 1.23, so it will have been present since then.

Reply:
Thank you for the clarification! I changed the references to Boundary controllers/workers to `0.18.x` and the Go version to `1.23`.